stracker/items/models.py

```diff
@@ -0,0 +1,76 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from core.models import BaseAbstractModel
+from customers.models import Customer
+
+
+class Category(BaseAbstractModel):
+    """
+    Model to hold item categories (types)
+    """
+
+    name = models.CharField(max_length=255, verbose_name=_("Category Name"))
+
+    class Meta:
+        verbose_name = _("category")
+        verbose_name_plural = _("categories")
+
+    def __str__(self):
+        return f"{self.name}"
+
+
+class Fridge(BaseAbstractModel):
+    """
+    Model to hold storage locations
+    """
+    name = models.CharField(verbose_name=_("name"), max_length=128, unique=True)
+
+    def __str__(self):
+        return f"{self.name}"
+
+
+class Shelf(BaseAbstractModel):
+    """
+    Model to hold storage divisions
+    """
+    name = models.CharField(verbose_name=_("name"), max_length=128, unique=True)
+    fridge = models.ForeignKey(Fridge, verbose_name=_("Fridge"), on_delete=models.CASCADE)
+    owner = models.ForeignKey(Customer,verbose_name=_("Owner"), on_delete=models.CASCADE)
+
+    def __str__(self):
+        return f"{self.name} - {self.fridge}"
+
+
+class Item(BaseAbstractModel):
+    """
+    Model to hold item details
+    """
+    sku = models.CharField(verbose_name=_("SKU"), max_length=128, unique=True)
+    name = models.CharField(verbose_name=_("name"), max_length=128)
+    owner = models.ForeignKey(Customer, verbose_name=_("Owner"), on_delete=models.PROTECT)
+    shelf = models.ForeignKey(Shelf, verbose_name=_("Shelf"), on_delete=models.CASCADE)
+    categories = models.ManyToManyField(Category, verbose_name=_("Categories"))
+    description = models.TextField(max_length=2000, verbose_name=_("Description"))
+    qrcode = models.ImageField(verbose_name=_("QR Code"))
```

Comment:
```suggestion
    qrcode = models.BinaryField(verbose_name=_("QR Code"))
```
qrcode'lar genel olarak text verisi içeriyor ama bunu binary olarak kodluyorlar. her kare bir bit olacak şekilde. dolayısıyla eğer textfield olarak tutmak istemezsek binaryfield (https://docs.djangoproject.com/en/4.0/ref/models/fields/#binaryfield) kullanabiliriz. image olandan daha sağlıklı olacağını düşünüyorum. daha sonra binary'den qr koda basit bir python kodu ile dönüştürülebilir. image quality loss olmaz ve başka qrlar ile compare edilebilir. örneğin ürünün qr'nın değişmesi durumda

Reply:
Qrcode'u db'de tutmak yerine istendiği zaman direkt olarak ürün detayının url'ini oluşturup göndermek, ya da önceden hazırlanmış (mesela) uuid'ler ile ilişkilendirilmiş qrCode'ları ürün dolaba kaldırılırken tabloya eklemek de iyi bir fikir olabilir gibi. Bir konuşalım bunu da.